content/CORETutorials/JailsPluginsVMs/Plugins/MinIOPlugin.md

```diff
@@ -0,0 +1,208 @@
+
+    a. Go to the MinIO web UI portal for the S3 service deployment and note existing configuration settings including users, groups, access keys, and all other MinIO settings.
+
+    b. Go to **Plugins** and locate the installed Minio plugin or go to **Jails** and locate the jail matching the **Jail Name** entered during [installation of the Minio plugin](#installing-the-minio-plugin).
+
+    c. Click <i class="material-icons" aria-hidden="true" title="Expand">chevron_right</i> to expand Minio details and management options.
+
+    d. Click <span class="material-icons">play_arrow</span>&nbsp;**START** to start the plugin (if needed) and then click <span class="material-icons">settings</span>&nbsp;**MANAGE** to go to the **MinIO Console** and log in.
+
+    e. Configure MinIO settings in the new deployment to match those of the existing deployment.
+
+4. Use the **MinIO Console** to manually recreate data buckets from the existing S3 service deployment in the Minio Plugin, including rules, versioning, quotas, and locks.
+    Bucket names do not need to be identical for this process, but your workflow may benefit from using identical names.
+
+### Migrating Data
+
+After installing and configuring the Minio plugin and installing the MinIO Client (`mc`), begin migrating data.
+
+1. From the Linux or WSL command line, enter the command `mc alias set NEWALIAS PATH ACCESSKEY SECRETKEY` for both the origin MinIO service deployment and the destination Minio plugin deployment.
+
+    {{< trueimage src="images/CORE/13.0/MinioClientSetAlias.png" alt="MinIO Client Set Alias" id="MinIO Client Set Alias" >}}
+
+2. Enter `mc mirror --preserve --watch SOURCE/BUCKET TARGET/BUCKET` to begin copying data to the Minio plugin deployment.
+
+    {{< trueimage src="images/CORE/13.0/MinioClientMirror.png" alt="MinIO Client Mirror" id="MinIO Client Mirror" >}}
+
+    The `mc mirror` operation with the `--watch` flag does not end when the transfer is completed, as it continually looks for new files added to the source.
+    The transfer rate also does not reach 0, as it is an average throughout the operation.
+
+3. Verify the transfer has completed successfully by comparing bucket size and items through the web UI of each deployment.
+    **Usage** size may vary slightly due to rounding differences between versions, but the total number of objects should be the same.
+
+    {{< trueimage src="images/CORE/13.0/MinioPluginVerifyBuckets.png" alt="Compare Buckets to Verify Completion" id="Compare Buckets to Verify Completion" >}}
+
+4. Use <kbd>CTRL+C</kbd> to end the `mc mirror` operation.
+5. Go to **Services** and toggle the **RUNNING** S3 service to **STOPPED**.
+6. If needed, edit the Minio plugin to reflect the correct port numbers using the process in [Configuring the Minio Plugin for Migration](#configuring-the-minio-plugin-for-migration), step one.
```

Comment:
Why is this a step if it is not needed? If it is optional then  just say it is optional. If they need to check this then drop the "if needed" from the step and make the action the first part of the step.
```suggestion
6. Edit the Minio plugin to reflect the correct port numbers using the process in [Configuring the Minio Plugin for Migration](#configuring-the-minio-plugin-for-migration), step one.
```